pkg/portfwd/client.go

```diff
@@ -130,14 +93,32 @@ func (g GrpcClientRW) Write(p []byte) (n int, err error) {
 func (g GrpcClientRW) Read(p []byte) (n int, err error) {
 	in, err := g.stream.Recv()
 	if err != nil {
-		if errors.Is(err, io.EOF) {
-			return 0, nil
-		}
 		return 0, err
 	}
-	if len(in.Data) == 0 {
-		return 0, nil
-	}
 	copy(p, in.Data)
 	return len(in.Data), nil
 }
+
+func (g GrpcClientRW) Close() error {
```

Comment:
same as the comment in the next file: consider implementing these functions using a pointer receiver.

Reply:
Done